### ISSUE TITLE
Has anyone generated (unofficial) Debian Installer Bullseye builds?

### ISSUE THREAD

**Issue by TBBle** [amouiche/qnap_mtd_resize_for_bullseye#32]
I'm about to give away my old TS-419P, and since I have run this script successfully, I thought the best state to leave it in would be to have the Debian Installer pre-flashed (so I can secure-erase the drives, and still leave it bootable with the new partition layout, to avoid leaving the new user to hit https://github.com/amouiche/qnap_mtd_resize_for_bullseye/issues/20#issuecomment-1063757818 when they try to install Debian or something).

I was going to follow the simple procedure at https://github.com/amouiche/qnap_mtd_resize_for_bullseye/issues/20#issuecomment-1061510263, but I was wondering before I do that if anyone has built appropriate d-i kernel and initrd, or if I should just flash https://ftp.debian.org/debian/dists/buster/main/installer-armel/current/images/kirkwood/network-console/qnap/ts-41x/ and let the next user install buster and then upgrade.

**Comment by vincele** [comment]
Hello, I think the donatee will either not trust you and reinstall from scratch wiping it all, or they will live with an installed debian. I don't think you need to have debian installer pre-setup.
Just give them the information about accessing it (network config, reset ssh password to known good state, etc.)

This could be closed, IMHO.

**Comment by TBBle** [comment]
Yeah, I gave the device away (to a work colleague) the week I asked the question. I did indeed [reflash the buster installer](https://github.com/amouiche/qnap_mtd_resize_for_bullseye/issues/33#issuecomment-1257716180) before giving it away. I secure-erased the disks anyway, so I'd have had to do a new fresh install of Debian to give them an installed Debian.

They never asked any follow-up questions, so I assume they were either able to install buster and upgrade, or flashed the Qnap stock firmware over the top.
